service/README.md

```diff
@@ -0,0 +1,33 @@
+# Service lifecycle
+
+Service, being a bit of an overloaded term, requires some
+specific care to explain the various components in the `service`
+package in UberFx.
+
+## Instantiation
+
+Generally, you create a service in one of two ways:
+
+* The builder pattern, e.g. `service.WithModules(...).Build()`
+* Calling `service.New()` directly.
+
+The former is generally much easier, and used in all the examples, but `New` is exported
```

Comment:
If you want to get more straightforward:
"The former is generally easier. We use the builder pattern in all examples, but…"